router/worker.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package router
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/go-vela/server/api"
+	"github.com/go-vela/server/router/middleware"
+	"github.com/go-vela/server/router/middleware/perm"
+	"github.com/go-vela/server/router/middleware/worker"
+)
+
+// WorkerHandlers is a function that extends the provided base router group
+// with the API handlers for worker functionality.
+//
+// POST   /api/v1/users
+// GET    /api/v1/workers
+// GET    /api/v1/workers/:worker
+// PUT    /api/v1/workers/:worker
+// DELETE /api/v1/workers/:worker
+func WorkerHandlers(base *gin.RouterGroup) {
+	// Workers endpoints
+	workers := base.Group("/workers")
+	{
+		workers.POST("", perm.MustPlatformAdmin(), middleware.Payload(), api.CreateWorker)
+		workers.GET("", api.GetWorkers)
```

Comment:
I noticed we aren't using the `perm.MustPlatformAdmin()` function here.

This means that any user within Vela will be able to capture this information 👍 

At this point, I can't come up with any good reasons why we wouldn't want to allow this so my comment here is more of a food for thought looking forward to the future.

If, at any point, we decide to embed more sensitive information into the `workers` table, we might have to restrict this to only being accessible by platform admins.

Reply:
That was my thought process as well. I'm open to changing it if we have concerns with other features we may want to add.

Reply:
My only concern with it not being plat admin would be some type of security attack vector. The only type I can really think of is some type of social engineering attack. None of the fields within that type stand out as something I'm super concerned with but I think depending on how that grows. Security would be my continued concern with not being a plat admin